robohive/envs/arms/push_base_v0.py

```diff
@@ -25,6 +30,27 @@ class PushBaseV0(env_base.MujocoEnv):
 
     def __init__(self, model_path, obsd_model_path=None, seed=None, **kwargs):
```

Comment:
Currently it looks like, the original env is being replaced by this new IK based env. Right? I think the right way to implement this is to 
1. define a new [act mode](https://github.com/vikashplus/robohive/blob/main/robohive/robot/robot.py#LL47C48-L47C52 ) called IK
2. Once the robot receives `ctrl_desired` in the step, it relies on the [process-actuator](https://github.com/vikashplus/robohive/blob/main/robohive/robot/robot.py#L673) call to do the translation from ee space to jnt space.
3. I also think the [action space definitions](https://github.com/vikashplus/robohive/blob/main/robohive/envs/env_base.py#LL95C9-L98C1) needs to be updated if its IK space

In summary, landing this in a generic way seems to require a rather larger refactor (possibly involving a refactor of the robot class). Let me know if should discuss this before going down this route.